doc/source/whatsnew/v1.0.0.rst

```diff
@@ -934,6 +934,7 @@ I/O
 - :meth:`read_gbq` now accepts ``progress_bar_type`` to display progress bar while the data downloads. (:issue:`29857`)
 - Bug in :func:`pandas.io.json.json_normalize` where a missing value in the location specified by `record_path` would raise a ``TypeError`` (:issue:`30148`)
 - :func:`read_excel` now accepts binary data (:issue:`15914`)
+- Bug in :func: 'to_pickle' and :func: 'read_pickle' where not accepting URL (:issue:'30163')
```

Comment:
where -> were

this is not a bug fix, move to other enhancements

Reply:
Enhancement does make a lot more sense. Updated.